sample_apps/run_example.sh

```diff
@@ -868,6 +959,28 @@ function get_measurement_of_trusted_simple_app_under_keystone() {
     get_measurement_of_app_by_name "keystone_example_app.exe"
 }
 
+# ###########################################################################
+# Currently, we hard-code the app's measurement. Eventually this routine
+# will want to look like the following, calling the common method
+# get_measurement_of_app_by_name to work on "cca_example_app.exe"
+# ###########################################################################
+function get_measurement_of_trusted_simple_app_under_cca() {
+    # FIXME: This is left here for future-use
+    # get_measurement_of_app_by_name "cca_example_app.exe"
+
+    run_cmd
+    local mrenclave="6190EB90B293886C172EC644DAFB7E33EE2CEA6541ABE15300D96380DF525BF9"
```

Comment:
Hard-coded measurement for now ... Please check , @jinbpark @bitboom

Reply:
This is equal to the measurement of the hard-coded report in ISLET SDK. You can use it as is.